File: src/Rotslyn.Transpilers.Tests/Samples/BasicClassWithGetterOnlyProperties.ts
module Rotslyn.Transpilers.Tests.Samples {
    export class BasicClassWithGetterOnlyProperties {
        private _intMember: number;
        public get intMember(): number {
            return this._intMember;
        }

        private _doubleMember: number;
        public get doubleMember(): number {
            return this._doubleMember;
        }

        private _doubleTypeMember: number;
        public get doubleTypeMember(): number {
            return this._doubleTypeMember;
        }

        private _floatMember: number;
        public get floatMember(): number {
            return this._floatMember;
        }

        private _floatTypeMember: number;
        public get floatTypeMember(): number {
            return this._floatTypeMember;
        }

        private _decimalMember: number;
        public get decimalMember(): number {
            return this._decimalMember;
        }

        private _decimalTypeMember: number;
        public get decimalTypeMember(): number {
            return this._decimalTypeMember;
        }

        private _int16Member: number;
        public get int16Member(): number {
            return this._int16Member;
        }

        private _int32Member: number;
        public get int32Member(): number {
            return this._int32Member;
        }

        private _int64Member: number;
        public get int64Member(): number {
            return this._int64Member;
        }

        private _stringKeywordMember: string;
        public get stringKeywordMember(): string {
            return this._stringKeywordMember;
        }

        private _stringTypeMember: string;
        public get stringTypeMember(): string {
            return this._stringTypeMember;
        }

        private _booleanKeywordMember: boolean;
        public get booleanKeywordMember(): boolean {
            return this._booleanKeywordMember;
        }

        private _booleanTypeMember: boolean;
        public get booleanTypeMember(): boolean {
            return this._booleanTypeMember;
        }
    }
}

File: src/Rotslyn.Transpilers.Tests/Samples/BasicClassWithGetterOnlyProperties.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Rotslyn.Transpilers.Tests.Samples
{
    public class BasicClassWithGetterOnlyProperties
    {
        public int IntMember { get; }
        public double DoubleMember { get; }
        public Double DoubleTypeMember { get; }
        public float FloatMember { get; }
        public Single FloatTypeMember { get; }
        public decimal DecimalMember { get; }
        public Decimal DecimalTypeMember { get; }
        public Int16 Int16Member { get; }
        public Int32 Int32Member { get; }
        public Int64 Int64Member { get; }
        public string StringKeywordMember { get; }
        public String StringTypeMember { get; }
        public bool BooleanKeywordMember { get; }
        public Boolean BooleanTypeMember { get; }
    }
}
